<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>728</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="year_widget">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>37</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>4</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>108</y>
     <width>16</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>109</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>17</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>250</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="name_1">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>121</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>성명</string>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>145</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="year_hun">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>년도 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>138</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="marks_1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>비고</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>헌금액</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hun_date">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>36</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>헌금일자</string>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>헌금명칭 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>146</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="marks_widget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hun_name_widget">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>72</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fix_hun_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>75</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>100</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>1</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addhun_input">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금등록</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>173</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_widget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>148</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="huncancel_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>닫기(저장취소)</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hunsave_button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hap_total">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록되는 헌금총액 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hap_total_widget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="hun_date_widget">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>35</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate|Qt::ImhPreferNumbers</set>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="month_hun">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>37</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>월 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_widget">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="week_hun">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>40</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>몇번째 주 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="week_widget">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sum_name_code_1">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>코드</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="registed_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>148</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="hun_detail_combo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>71</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="Bankincome_check">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>76</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="locale">
    <locale language="Kinyarwanda" country="Rwanda"/>
   </property>
   <property name="text">
    <string>통장예입</string>
   </property>
   <property name="autoRepeatDelay">
    <number>302</number>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>127</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(단위:천원)</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="hun_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>개별코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>성명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>통장예입</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="re_calculate_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>합계금액 재계산</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="code1_widget">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>155</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>35</y>
     <width>65</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_name_combo_widget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>36</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_row_button">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행 삭제</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>year_widget</tabstop>
  <tabstop>month_widget</tabstop>
  <tabstop>week_widget</tabstop>
  <tabstop>gubun_name_combo_widget</tabstop>
  <tabstop>hun_name_widget</tabstop>
  <tabstop>hun_detail_combo</tabstop>
  <tabstop>registed_name</tabstop>
  <tabstop>amount_widget</tabstop>
  <tabstop>marks_widget</tabstop>
  <tabstop>addhun_input</tabstop>
  <tabstop>hunsave_button</tabstop>
  <tabstop>hun_tableWidget</tabstop>
  <tabstop>re_calculate_button</tabstop>
  <tabstop>remove_row_button</tabstop>
  <tabstop>Bankincome_check</tabstop>
  <tabstop>huncancel_button</tabstop>
  <tabstop>hun_date_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>addhun_input</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hun_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hunsave_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>643</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>huncancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>re_calculate_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>re_calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_row_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>remove_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hun_input()</slot>
  <slot>file_save()</slot>
  <slot>re_calculate()</slot>
  <slot>file_close()</slot>
  <slot>remove_row()</slot>
 </slots>
</ui>
